package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "audion",
+  "version": "1.0.0",
+  "description": "Traces web audio API calls.",
+  "main": "index.js",
+  "engines": {
+    "node": "14"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.14.6",
+    "@babel/plugin-transform-modules-commonjs": "^7.14.5",
+    "@pixi/jsdoc-template": "^2.6.0",
+    "babel-jest": "^27.0.6",
```

Comment:
I guess the Jest test framework requires Babel?

Reply:
It's one option. `babel-jest` provides the means to use babel on files jest loads. Another option would be to use node's module support but I think that would be more trouble.